certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/Main.kt

```diff
@@ -123,18 +120,48 @@ class Main : Callable<Int> {
 
   private fun queryHost() {
     val x509certificates = fromHttps(host!!)
-    prettyPrintChain(x509certificates)
 
-    // TODO We should add SANs and complete wildcard hosts.
-    addHostToCompletionFile(host!!)
+    if (x509certificates.isEmpty()) {
+      System.err.println("Warn: ${Ansi.AUTO.string(" @|yellow No trusted certificates|@")}")
+    }
+
+    val output = output
 
-    if (output != null) {
-      try {
-        outputCertificates(output!!, x509certificates)
-      } catch (ioe: IOException) {
-        throw UsageException("Unable to write to $output", ioe)
+    x509certificates.forEachIndexed { i, certificate ->
+      if (i > 0) {
+        println()
```

Comment:
?

Reply:
Just makes it clearer that the certificate PEM belongs with which set of factual statements (common name etc).

Reply:
I don't follow. If we're not printing anything, why not just add a comment?

Reply:
We always at least print on line 160, and conditionally on 148.

Reply:
Oh, because it's CLI we wanna print a new line? All good.